src/pages/call/call.html

```diff
@@ -1,7 +1,53 @@
 <ion-header>
-	<header [userData]="userData"></header>
+    <header [userData]="userData"></header>
 </ion-header>
 
 <ion-content padding>
-	Call content
+
+    <ion-grid text-center>
+        <ion-row>
+            <ion-col>
+                <img src="assets/img/addcontact_btn.svg">
+            </ion-col>
+            <ion-col>
+                <h1>&nbsp;{{phoneNumber}}&nbsp;</h1>
+            </ion-col>
+            <ion-col>
+                <img src="assets/img/delete_btn.svg"
+                     (press)="removeAll()" (click)="removeLast()">
+            </ion-col>
+        </ion-row>
```

Comment:
You can replace these images with [ionic icons](https://ionicframework.com/docs/ionicons/).
 And by the way checkout [ionic grid](https://ionicframework.com/docs/api/components/grid/Grid/) system, it is pretty useful for this case.
if you add a the directive `col-8` on the phone number element and `col-2` on the buttons around it, it will prevent changing the position of the buttons when user add a long number
Kinda like this
```html
<ion-row class="phone-number-wrapper">
     <ion-col col-2 align-items-center>
         <ion-icon [hidden]="!phoneNumber.length" ios="ios-person-add" md="md-person-add"></ion-icon>
         </ion-col>
         <ion-col col-8 class="phone-number">
             <h1>{{phoneNumber}}</h1>
          </ion-col>
        <ion-col col-2 align-items-center>
          <ion-icon [hidden]="!phoneNumber.length" ios="ios-backspace" md="md-backspace" (press)="removeAll()" (tap)="removeLast()"></ion-icon>
     </ion-col>
 </ion-row>
```
scss
```scss
page-call {
    .phone-number {
        flex: 1;
        display: flex;
        align-items: center;
        justify-content: center;
    }
    .phone-number-wrapper {
        height: 60px;
    }
}
```